pkg/tangy/queries.go

```diff
@@ -0,0 +1,33 @@
+package tangy
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/jackc/pgx/v5"
+	"golang.org/x/exp/rand"
+)
+
+func contentIdsInVersion(repoId string, versionNum int, namedArgs *pgx.NamedArgs) string {
+	ran := rand.Int()
+	repoIdName := fmt.Sprintf("%v%v", "repoName", ran)
+	versionNumName := fmt.Sprintf("%v%v", "versionNum", ran)
```

Comment:
can you put a comment above the function saying what the function is supposed to do?

Reply:
can you also put a clarifying one for why randomly generating a repoName and versionName is needed?